src/Symfony/Component/Serializer/Tests/Fixtures/Attributes/IgnoreDummy.php

```diff
@@ -28,4 +28,16 @@ public function getIgnored2()
     {
         return $this->ignored2;
     }
+
+    #[Ignore]
+    public function getSomethingElse(): int
+    {
+        return 42;
+    }
```

Comment:
This additional test case seems not related to your PR topic. Is there a use for it that I've missed?

Reply:
`ignored2` is itself a property, I wanted to make sure it also works if its just the accessor, without any obvious property backing it. But I'll remove it.